### PR TITLE
fix: safe remove()

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Fix a bug that caused the AviUtl folder to be deleted when a package containing ` { "filename": "", "isObsolete": true },` is installed or a package containing ` { "filename": "" },` is uninstalled.

This bug currently only occurs when loading incorrectly configured package data. It has never occurred in normal use.

## How Has This Been Tested?

- Installation or uninstallation of the package containing the above settings fails.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/renderer/main/package.js [hunk 1]
```diff
@@ -792,7 +792,9 @@ async function uninstallPackage(instPath) {
     if (!file.isInstallOnly)
       filesToRemove.push(path.join(instPath, file.filename));
   }
-  await Promise.all(filesToRemove.map((filePath) => remove(filePath)));
+  await Promise.all(
+    filesToRemove.map((filePath) => safeRemove(filePath, instPath))
+  );
```
Comment (hal-shu-sato) [inline src/renderer/main/package.js:797]: Could you add try...catch around here?
It seems that I forgot to add.

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
Sorry.
I need your help on one point.

File: src/renderer/main/package.js (hunk 1 above)
Comment (mitosagi) [reply src/renderer/main/package.js]: I had forgotten too.

Fixed.

File: src/renderer/main/package.js [hunk 2]
```diff
@@ -792,9 +792,18 @@ async function uninstallPackage(instPath) {
...
-  await Promise.all(
-    filesToRemove.map((filePath) => safeRemove(filePath, instPath))
-  );
+
+  try {
+    await Promise.all(
+      filesToRemove.map((filePath) => safeRemove(filePath, instPath))
+    );
+  } catch {
+    buttonTransition.message(btn, 'エラーが発生しました。', 'danger');
```
Comment (hal-shu-sato) [inline src/renderer/main/package.js:801]:
```suggestion
  } catch (e) {
    log.error(e);
    buttonTransition.message(btn, 'エラーが発生しました。', 'danger');
```
Please log all caught errors.

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
I have the last part to want to be fixed.

File: src/renderer/main/package.js (hunk 2 above)
Comment (mitosagi) [reply src/renderer/main/package.js]: Fixed.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for fixing this problem!

**Pull request merged**